tests/daily_scraper/test_daily_scraper.py

```diff
@@ -305,11 +316,20 @@ def test_scraper(self):
         """Testing various download scenarios for DailyScraper"""
 
         for entry in tests:
-            scraper = DailyScraper(directory=self.temp_dir, base_url=self.wdir,
+            scraper = DailyScraper(destination=self.temp_dir, base_url=self.wdir,
                                    version=None, log_level='ERROR',
                                    **entry['args'])
             expected_target = os.path.join(self.temp_dir, entry['target'])
             self.assertEqual(scraper.target, expected_target)
+            destination_ext = DEFAULT_FILE_EXTENSIONS[entry['args']['platform']]
+            if 'extension' in entry['args']:
+                destination_ext = entry['args']['extension']
+            destination = os.path.join(self.temp_dir,"file." + destination_ext)
+            scraper2 = DailyScraper(destination=destination, base_url=self.wdir,
+                                   version=None, log_level='ERROR',
+                                   **entry['args'])
+            expected_target = destination
+            self.assertEqual(scraper2.target, expected_target)
```

Comment:
Please move those additions out to its own test module, which is specifically about testing the `--destination` command line option.